gpu-operator/security.rst

```diff
@@ -0,0 +1,103 @@
+
+*****************************
+Security Considerations
+*****************************
+
+
+Pod Security Context of the Operator and Operands
+=================================================
+
+Several of the NVIDIA GPU Operator operands, such as the driver containers and container toolkit,
+require the following elevated privileges:
+
+- ``privileged: true``
+- ``hostPID: true``
+- ``hostIPC: true``
+
+The elevated privileges are required for the following reasons:
+
+- Access to the host file system and hardware devices, such as NVIDIA GPUs.
+- Restart system services such as containerd.
+- Permit users to list all GPU clients using the ``nvidia-smi`` utility.
+
+Only the Kubernetes cluster administrator needs to access or manage the Operator namespace.
+As a best practice, establish proper security policies and prevent any other users from accessing the Operator namespace.
+
+
+
+CVEs

```

Comment:
Do we think a table like this is useful? Are their other fields we should maintain in our docs (all the CVE info is also in the linked bulletins)

There are a few more of these CVEs that relate to vulnerabilities in images used in older version. For completeness, we should probably include them as well, but i wanted to make sure that the table was valuable before i went to the trouble of adding all the rest.

Reply:
I think it is valuable. I don't think we should add any more information to this table though. The security bulletin should be the source of truth and readers should be redirected there if they want more information about any particular CVE.